modules/renter/proto/contract.go

```diff
@@ -29,6 +29,11 @@ type updateSetHeader struct {
 	Header contractHeader
 }
 
+type v132UpdateSetHeader struct {
```

Comment:
we need a docstring, ideally indicating as well what went wrong with the old thingy and why we chose to upgrade